test/inspector/inspector-helper.js

```diff
@@ -427,9 +427,25 @@ Harness.prototype.expectShutDown = function(errorCode) {
   });
 };
 
-exports.startNodeForInspectorTest = function(callback) {
-  const child = spawn(process.execPath,
-      [ '--inspect-brk', mainScript ]);
+Harness.prototype.kill = function() {
+  return this.enqueue_((callback) => {
+    this.process_.kill();
+    callback();
+  });
+};
+
+exports.startNodeForInspectorTest = function(callback, opt_skip_break,
+                                             opt_script_contents) {
```

Comment:
A default param might be nicer for the second argument, i.e., `function(callback, arg = '--inspect-brk', contents)`.  The string `'--inspect'` makes the intent clearer at the call site than a boolean does.

Style issue: we generally use camelCase for arguments and locals in JS land.